cmd/soroban-rpc/internal/methods/send_transaction.go

```diff
@@ -21,6 +21,9 @@ type SendTransactionResponse struct {
 	// ErrorResultXDR is a TransactionResult xdr string which contains details on why
 	// the transaction could not be accepted by stellar-core.
 	ErrorResultXDR string `json:"errorResultXdr,omitempty"`
+	// DiagnosticEventsXDR is present only if Status is equal to proto.TXStatusError.
+	// DiagnosticEventsXDR is a base64-encoded slice of xdr.DiagnosticEvent
+	DiagnosticEventsXDR []string `json:"diagnosticEventsXdr,omitempty"`
```

Comment:
is there a separate process initiated as part of rpc api resource changes, i.e. the associated soroban-docs updates and sdk blast/mention on this new field so they can handle? such as for js-stellar-sdk [SendTransactionResponse](https://github.com/stellar/js-stellar-sdk/blob/master/src/soroban/api.ts#L187-L207)

Reply:
I will create a ticket in this repo and the docs repo once this is merged